gnovm/tests/files/zrealm_crossrealm29.gno

```diff
@@ -0,0 +1,35 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	"gno.land/r/demo/tests/crossrealm_b"
+)
+
+type Struct struct {
+	A int
+}
+
+var s *Struct
+
+func init() {
+	s = &Struct{A: 100}
+	// This is OK, 's' is short for 'crossrelm_test.s',
+	// so there are no captures for the closure.
+	// Since s is a 'field' of the package block
+	// the object 's' is not stored in crossrealm_b,
+	// it is lazily attached to this realm!!!
+	withswitch(crossrealm_b.SetObject)(func() { &s.A })
```

Comment:
a bit surprising, but makes sense.
see zrealm_crossrealm28.gno for a similar example that fails.
what this means is that you can "attach" an unreal object by 
passing in a closure that returns the value if it is derived from
another package's global vars. 
poor man's attachment.